.nuke/parameters.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "./build.schema.json",
+  "Solution": "source/AzureDevOpsIssueTracker.sln"
```

Comment:
nit: does this need a ./ prefix as per line-2?

Reply:
I don't think so. None of the others have it, and Nuke is .NET based, so pretty sure won't be picky about the leading `./` on that path. `$schema` is different, which does seem odd when they're right next to each other.